atum/src/main/scala/za/co/absa/atum/AtumImplicits.scala

```diff
@@ -260,6 +261,40 @@ trait AtumImplicitsBase {
       dataset
     }
 
+    /**
+     * The method returns ControlMeasure object from the Atum context
+     * @return - ControlMeasure object
+     */
+    def getControlMeasure: ControlMeasure = {
+      atum.preventNotInitialized()
+      atum.controlFrameworkState.accumulator.getControlMeasure
+    }
+
+    /**
+     * The method returns AditionalInfo object from the Atum context
+     * @return - AditionalInfo object
+     */
+    def getAllAdditionalInfo: Map[String, String] = {
+      atum.preventNotInitialized()
+      atum.controlFrameworkState.accumulator.getControlMeasure.metadata.additionalInfo
+    }
```

Comment:
```suggestion
     * The method returns AdditionalInfo object from the Atum context
     * @return - AdditionalInfo object
     */
    def getAllAdditionalInfo: Map[String, String] = {
      atum.preventNotInitialized()
      atum.controlFrameworkState.accumulator.getControlMeasure.metadata.additionalInfo
    }
```

While I understand the usefulness of the `getControlMeasure` accessor, I'd consider this accessor a bit unnecessary, there is little added value over calling `getControlMeasure.metadata.additionalInfo`, I think.